config/locales/en.yml

```
@@ -44,6 +44,8 @@ en:
         state: State
         source: Source
       spree/product:
+        name: "Product Name"
+        price: "Product Price"
```

I checked what we use in the form where the errors appear and it's "Product Name" and just "Price". It would be good to be consistent here.
```suggestion
        name: "Product Name"
        price: "Price"
```

---

Hi @mkllnk Thank you for your comments.
- I changed "Product Price" to "Price" in en.yml
- Unit value correspond to variant_unit in the yml, which was an existing key in the yml.  May be the translation is not working very well? I changed the corresponding to simply "Unit" (was "Variant Unit") to see it will change the traduction for other langages. But I can't reproduce localy the Transifex translation, can I ?